cpp/core/compute/Runtime.h

```diff
@@ -42,8 +42,9 @@ struct SparkTaskInfo {
   // Same as TID.
   int64_t taskId{0};
 
-  std::string toString() const {
```

Comment:
Thanks for your PR!
I think it may be better to keep `toString`. Then it can be called from other places, as well as from the implemented `<<` operator . Right?

Reply:
I keep toString and add new overload `<<` operator.